src/components/editor/share-modal/style.css

```diff
@@ -1,8 +1,14 @@
 .editor-share-modal__links {
-	display: grid;
-	grid-template-columns: 1fr 1fr;
-	grid-gap: calc(var(--grid-unit) * 2);
-	align-items: top;
+  display: grid;
+  grid-template-rows: 1fr;
```

Comment:
Feels like we need a grid-gap here as the write/read areas are a bit too close.

Reply:
Ah, good catch! Updating.

Reply:
Must look better with ed39282b36a3a6e2b65dc70dfcaf24d9eb0e5d15.